tests/test_cli.py

```diff
@@ -61,6 +61,22 @@ def test_extract_command_with_gelf_parser(gelf_logger):
         assert '{"username": "foo"}' in result.output
 
 
+def test_verbosity_option_should_impact_logging_behaviour():
+    """Test that the verbosity option impacts the extract command logging"""
+
+    runner = CliRunner()
+    result = runner.invoke(cli, ["-v", "CRITICAL", "extract", "-p", "gelf"], input=None)
+    assert result.output == ""
+    result = runner.invoke(cli, ["-v", "ERROR", "extract", "-p", "gelf"], input=None)
+    assert result.output == ""
+    result = runner.invoke(cli, ["-v", "WARNING", "extract", "-p", "gelf"], input=None)
+    assert result.output == ""
+    result = runner.invoke(cli, ["-v", "INFO", "extract", "-p", "gelf"], input=None)
+    assert "Extracting events using the gelf parser" in result.output
+    result = runner.invoke(cli, ["-v", "DEBUG", "extract", "-p", "gelf"], input=None)
+    assert "Extracting events using the gelf parser" in result.output
```

Comment:
Should be better to have an extra message to test the debug level?

Reply:
Ok, I will give it a try)

Reply:
Updated